sdk/core/azure-core-http-netty/README.md

```diff
@@ -34,6 +34,8 @@ The following sections provide several code snippets covering some of the most c
 Create a Netty Http client that uses port 80 and has no proxy.
 
 ```java
+// ./src/samples/java/com/azure/core/http/netty/ReadmeSamples.java#L23-L23
```

Comment:
I agree with Alan, I'm not convinced that we want these to show up in all of the code samples. If we can hide it, whilst still retaining the ability for it to be updated when the sample code is changed, that would be best IMHO.

Reply:
Using alternate embedding syntax that will hide the comment in README and will continue to be updated when sample code changes.